lib/registration.pm

```diff
@@ -87,6 +87,7 @@ our %ADDONS_REGCODE = (
     'sle-live-patching'        => get_var('SCC_REGCODE_LIVE'),
     'SLES-LTSS'                => get_var('SCC_REGCODE_LTSS'),
     'SUSE-Linux-Enterprise-RT' => get_var('SCC_REGCODE_RT'),
+    ESPOS                      => get_var('SCC_REGCODE_ESPOS'),
```

Comment:
ESPOS -> 'ESPOS'

Reply:
> ESPOS -> 'ESPOS'
If add single quotes, it would print errors when auto checking.